src/nodes/core/BypassNode.js

```diff
@@ -1,6 +1,11 @@
 import Node from './Node.js';
 import { addMethodChaining, nodeProxy } from '../tsl/TSLCore.js';
 
+/**
+ * TODO: Explain the purpose of this module.
```

Comment:
`BypassNode` generates the code of a node but returns another node in the output. This can be used to call a method or node that does not return a value, i.e. type `void` on an input where returning a value is required. Example: `material.colorNode = myColor.bypass( runVoidFn() )`